lightspeed/modules/proc_update-connection-secrets.adoc

```diff
@@ -0,0 +1,31 @@
+:_content-type: PROCEDURE
+
+[id="update-connection-secrets_{context}"]
+
+= Updating connection secrets on an existing {PlatformName} operator
+
+After you have set up the {LightspeedShortName} on-premise deployment successfully, you can modify the deployment if you want to connect to another {ibmwatsonxcodeassistant} model. For example, you connected to the default {ibmwatsonxcodeassistant} model but now want to connect to a custom model instead. To connect to another {ibmwatsonxcodeassistant} model, you must create new connection secrets, and then update the connection secrets and parameters on an existing {PlatformName} operator.
+
+.Prerequisites
+* You have set up a {LightspeedShortName} on-premise deployment. 
+* You have obtained an API key and a model ID of the {ibmwatsonxcodeassistant} model you want to connect to. 
+* You have created new authorization and model connection secrets for the {ibmwatsonxcodeassistant} model that you want to connect to. For information about creating authorization and model connection secrets, see xref:create-connection-secrets_configuring-lightspeed-onpremise[Creating connection secrets].
+
+.Procedure
+. Go to the {OCP}. 
+. Select *Operators → Installed Operators*.
+. From the *Projects* list, select the namespace where you installed the {LightspeedShortName} instance.
+. Locate and select the *Ansible Automation Platform (provided by Red Hat)* operator that you installed earlier.
```

Comment:
I'd use the {PlatformNameShort} attribute here.

Reply:
This is the UI label so not making any update here.